cmd/dump.go

```diff
@@ -35,6 +35,11 @@ It can be used for backup and capture Gitea server image to send to maintainer`,
 			Value: "custom/conf/app.ini",
 			Usage: "Custom configuration file path",
 		},
+		cli.StringFlag{
+			Name:  "file, f",
```

Comment:
Kindly update the docs to reflect the new flag

Reply:
done.